### DISCUSSION TITLE
Resource request and limits for OpenDJ container in production environement

### DISCUSSION THREAD

**Discussion by amareshnayak822** [OpenIdentityPlatform/OpenDJ#162] [Q&A]
We are deploying OpenDJ in kubernetes production environment cluster.Although it is highly depends upon the traffic still can anyone suggest how much "minimum cpu and memory requirement for opendj container" to run in prod env cluster.The limits can be decided based on the traffic but minimum resource request values are needed to run OpenDJ container.

Can any one suggest?

**Comment by vharseko** [comment]
--memory="1g"
